tests/integrationTests/tests/cpp_count_ts/__init__.py

```diff
@@ -0,0 +1,87 @@
+# Necessary imports. Provides library functions to ease writing tests.
+from lib import prebuild, testcase, SUBMITTY_INSTALL_DIR
+
+import subprocess
+import os
+import glob
+import shutil
+import traceback
+
+############################################################################
+# COPY THE ASSIGNMENT FROM THE SAMPLE ASSIGNMENTS DIRECTORIES
+
+SAMPLE_ASSIGNMENT_CONFIG = SUBMITTY_INSTALL_DIR + "/more_autograding_examples/cpp_count_ts/config"
+SAMPLE_SUBMISSIONS       = SUBMITTY_INSTALL_DIR + "/more_autograding_examples/cpp_count_ts/submissions"
+
+@prebuild
+def initialize(test):
+    try:
+        os.mkdir(os.path.join(test.testcase_path, "assignment_config"))
+    except OSError:
+        pass
+    try:
+        data_path = os.path.join(test.testcase_path, "data")
+        if os.path.isdir(data_path):
+            shutil.rmtree(data_path)
+        os.mkdir(data_path)
+    except OSError:
+        pass
```

Comment:
What is the intended purpose of these try-except statement?

Reply:
These files are not cleaned up when a test is finished since they are not specific to test runs.

Reply:
Yes that makes sense. Updated the code